cmd/agent/app/integrations.go

```diff
@@ -268,12 +318,25 @@ func installTuf(cmd *cobra.Command, args []string) error {
 	}
 
 	intVer := strings.Split(args[0], "==")
-	integration := strings.TrimSpace(intVer[0])
+	integration := strings.Replace(strings.TrimSpace(intVer[0]), "_", "-", -1)
 	if integration == "datadog-checks-base" {
-		return fmt.Errorf("cannot upgrade datadog-checks-base")
+		return fmt.Errorf("this command does not allow installing datadog-checks-base")
+	}
+	versionToInstall, err := parseVersion(strings.TrimSpace(intVer[1]))
+	if err != nil {
+		return fmt.Errorf("unable to get version of %s to install: %v", integration, err)
 	}
-	versionToInstall := strings.TrimSpace(intVer[1])
-	currentVersion, err := getIntegrationVersion(integration, cachePath)
+	minVersion, err := minAllowedVersion(integration)
+	if err != nil {
+		return fmt.Errorf("unable to get minimal version of %s: %v", integration, err)
+	}
+	if !versionToInstall.isAbove(minVersion) {
```

Comment:
Right now, this function causes an unnecessary installation to happen when `versionToInstall == minAllowedVersion`, right? Is that what we want?

Either that, or we gotta check whether `versionToInstall == currentVersion` below.